sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/BaseClause.cs

```diff
@@ -1,10 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Azure.DigitalTwins.Core.QueryBuilder
```

Comment:
Consider changing all the base classes like this `BaseClause` or `BaseCondition` to `ClauseBase` or `ConditionBase` as they are base classes for those types ... it's nitty but hopefully makes sense.